indico/modules/events/sessions/templates/management/types_dialog.html

```diff
@@ -0,0 +1,103 @@
+{% from 'message_box.html' import message_box %}
+{% from 'events/sessions/management/_types_table.html' import types_table_row %}
+
+<div class="manage-session-types">
+    <div class="toolbar space-after hide-if-locked">
+        <button id="js-new-session-type" class="i-button icon-plus highlight" data-ajax-dialog
```

Comment:
I would `js-new-session-type` move to the `class` attribute since this is what we have been doing so far ;)

Reply:
:+1: on that; also move the `data-ajax-dialog` at the end (we put data attrs with no value last)